koa-compress/koa-compress.d.ts

```diff
@@ -0,0 +1,41 @@
+// Type definitions for koa-compress v2.x
+// Project: https://github.com/koajs/compress
+// Definitions by: Jerry Chin <https://github.com/hellopao/>
+// Definitions: https://github.com/hellopao/DefinitelyTyped
+
+/* =================== USAGE ===================
+
+    import compress = require("koa-compress");
+    var Koa = require('koa');
+
+    var app = new Koa();
+    app.use(compress());
+
+ =============================================== */
+/// <reference path="../node/node.d.ts" />
+/// <reference path="../koa/koa.d.ts" />
+
+declare module "koa-compress" {
+
+    import * as Koa from "koa";
+    import * as zlib from "zlib";
+
+    interface ICompressOptions extends zlib.ZlibOptions {
```

Comment:
http://www.typescriptlang.org/docs/handbook/writing-declaration-files.html

> Naming Conventions
In general, you shouldn’t prefix interfaces with I (e.g. IColor). Because the concept of an interface in TypeScript is much more broad than in C# or Java, the IFoo naming convention is not broadly useful.